code/modules/religion/religion_sects.dm

```diff
@@ -42,18 +42,35 @@
 	var/smack_chance = DEFAULT_SMACK_CHANCE
 	/// Whether the structure has CANDLE OVERLAYS!
 	var/candle_overlay = TRUE
+	///people who have agreed to join the crusade, and can be deaconized
+	var/list/possible_crusaders = list()
+	///people who have been offered an invitation, they haven't finished the alert though.
+	var/list/currently_asking = list()
 
 /datum/religion_sect/New()
 	. = ..()
 	if(desired_items)
 		desired_items_typecache = typecacheof(desired_items)
+	if(!locate(/datum/religion_rites/deaconize) in rites_list)
+		rites_list += list(/datum/religion_rites/deaconize/one_time_use)
 	on_select()
 
 /// Activates once selected
 /datum/religion_sect/proc/on_select()
 	SHOULD_CALL_PARENT(TRUE)
 	SSblackbox.record_feedback("text", "sect_chosen", 1, name)
 
+/**
+ * Called by deaconize rite, this async'd proc waits for a response on joining the sect.
+ * If yes, the deaconize rite can now recruit them instead of just offering invites
+ */
+/datum/religion_sect/proc/invite_crusader(mob/living/carbon/human/invited, crusader)
+	currently_asking += invited
+	var/ask = tgui_alert(invited, "Join [GLOB.deity]?[crusader ? " You will be bound to a code of honor." : " You will be expected to follow the Chaplain's order."]", "Invitation", list("Yes", "No"), 60 SECONDS)
```

Comment:
Deaconization being a subtype of honorbound's rite feels bad, especially the crusader var. Is there no better way we could do this? Maybe actually invert them, and make the honorbound one a subtype of deaconization with string overrides instead of a snowflake var?

Reply:
It would still require snowflake stuff for the effects of ``invoke_effect`` to remove the rite

Reply:
Make one-time rites be a flag/var, and swap the types around with the subtype not having it? Also invite_crusader has no right to be on the sect type, it could very well be on the ritual side and be overriden so you don't have to deal with the ``crusader`` var